docs/c_gpu.rst

```diff
@@ -221,15 +221,15 @@ For type 1 these points are "sources", but for type 2, "targets".
 
     Input/Output:
 
-    plan        the cufinufft plan object from the above plan stage
+    plan        the plan object from the above plan stage
 
     Returns:
 
     status      zero if success, otherwise an error occurred
 
-Note: The user must not change the contents of the GPU arrays ``x``, ``y``, or ``z`` (in 3D case) between this step and the below execution step. They are read in the execution step also.
+Note: The user must not change the contents of the GPU arrays ``x``, ``y``, or ``z`` between this step and the below execution step. They are read in the execution step also.
 
-Note: The actual plan (not its pointer is passed in); new GPU arrays are allocated and filled in the internal plan struct that the plan points to.
+Note: The actual plan (not its pointer) is passed in; new GPU arrays are allocated and filled in the internal plan struct that the plan points to.
```

Comment:
this sentence is cryptic. do we understand it?

Reply:
No I was trying to salvage it. I think the first part is just contrasting with `makeplan` where we pass a pointer to a plan (handle) and here we're just passing the handle. The second part I don't know whether it's actually needed.

Reply:
Took a stab at fixing this.